.changeset/smol-kittens-blep.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/telemetry-utils": minor
+---
+
+Deprecate logIfFalse
+
+This functionality was not intended for export and will be removed in a future release.
+No replacement API is offered, as the logic is trivial to reproduce as needed.
```

Comment:
Should you mention that the MultiSinkLoggerProperties interface is now exported?